pkg/client/client.go

```diff
@@ -632,6 +632,9 @@ func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, sd StatusDetai
 			}
 			xdp = np.Acceleration
 			mode = np.Mode
+			if mode == models.KubeProxyReplacementFeaturesNodePortModeDSR {
+				dsrMode = np.DsrMode
+			}
```

Comment:
```suggestion
			if mode == models.KubeProxyReplacementFeaturesNodePortModeDSR ||
			mode == models.KubeProxyReplacementFeaturesNodePortModeHybrid
			{
				dsrMode = np.DsrMode
			}
```

Hybrid does a mix of DSR and SNAT, so the DSR-dispatch mode also becomes relevant here.

Reply:
Updated.